src/Discovery/Discovery.tsx

```diff
@@ -0,0 +1,552 @@
+                  ev.stopPropagation();
+                  setSelectedTags({
+                    ...selectedTags,
+                    [name]: selectedTags[name] ? undefined : true,
+                  });
+                }}
+              >
+                {name}
+              </Tag>
+            );
+          })}
+        </React.Fragment>
+      ),
+    },
+  );
+  if (config.features.authorization.enabled) {
+    columns.push({
+      title: 'Access',
+      render: (_, record) => (
+        record[accessibleFieldName]
+          ? (
+            <Popover
+              overlayClassName='discovery-table__access-popover'
+              placement='topRight'
+              arrowPointAtCenter
+              title={'You have access to this study.'}
+              content={<div className='discovery-table__access-popover-text'>
+                <>You have <code>{ARBORIST_READ_PRIV}</code> access to</>
+                <><code>{record[config.minimal_field_mapping.authz_field]}</code>.</>
+              </div>}
+            >
+              <UnlockOutlined className='discovery-table__access-icon' />
+            </Popover>
+          )
+          : (
+            <Popover
+              overlayClassName='discovery-table__access-popover'
+              placement='topRight'
+              arrowPointAtCenter
+              title={'You do not have access to this study.'}
+              content={
+                <div className='discovery-table__access-popover-text'>
+                  <>You don&apos;t have <code>{ARBORIST_READ_PRIV}</code> access to</>
+                  <><code>{record[config.minimal_field_mapping.authz_field]}</code>.</>
+                </div>
+              }
+            >
+              <LockFilled className='discovery-table__access-icon' />
+            </Popover>
+          )
+      ),
+    });
+  }
+  // -----
+
+  const handleSearchChange = (ev) => {
+    const value = ev.currentTarget.value;
+    setSearchTerm(value);
+    if (value === '') {
+      setSearchFilteredResources(props.studies);
+      return;
+    }
+    if (!jsSearch) {
+      return;
+    }
+    const results = jsSearch.search(value);
+    setSearchFilteredResources(results);
+  };
+
+  const handleAccessLevelChange = (ev) => {
+    const value = ev.target.value as AccessLevel;
+    setAccessLevel(value);
+  };
+
+  const visibleResources = filterByTags(
+    filterByAccessLevel(searchFilteredResources, accessLevel, accessibleFieldName),
+    selectedTags,
+  );
+
+  return (<div className='discovery-container'>
+    { (config.features.page_title && config.features.page_title.enabled) &&
+      <h1 className='discovery-page-title'>{config.features.page_title.text || 'Discovery'}</h1>
+    }
+    <div className='discovery-header'>
+      <div className='discovery-header__stats-container'>
+        {
+          config.aggregations.map(aggregation => (
+            <div className='discovery-header__stat' key={aggregation.name}>
```

Comment:
Optional suggestion: I think these stat boxes would benefit from vertical line separators on their right side, following the convention of the Data Explorer's stat counts.

Reply:
I think you're right! I'm adding it.
<img width="1440" alt="Screen Shot 2021-02-12 at 9 13 00 AM" src="https://user-images.githubusercontent.com/18353462/107785757-99aca700-6d12-11eb-9fae-1ae393c159e3.png">